controlplane/internal/controllers/rke2controlplane_controller.go

```diff
@@ -339,11 +339,13 @@ func (r *RKE2ControlPlaneReconciler) updateStatus(ctx context.Context, rcp *cont
 	}
 
 	if len(ownedMachines) == 0 {
-		return fmt.Errorf("no Control Plane Machines exist for RKE2ControlPlane %s/%s", rcp.Namespace, rcp.Name)
+		r.Log.Info(fmt.Sprintf("no Control Plane Machines exist for RKE2ControlPlane %s/%s", rcp.Namespace, rcp.Name))
```

Comment:
Please use `logger` defined at the beginning of this method. We need to clean up our logging, it a bit messy right now. It will already include name and namespace of the object.

Reply:
updated in 4a78a8a3204fbe45a8a0320f44f6f9861b908c8c